frontend/src/app/features/work-packages/components/op-wp-quick-add-modal/op-wp-quick-add-modal.html

```diff
@@ -0,0 +1,18 @@
+<div
```

Comment:
I think I'd prefer a form with an `ngSubmit` handler, but I'm not sure how the others see this.

Reply:
I've talked to @psatyal, and we've agreed to make this as standard a modal as possible, which means making this a form and adding cancel & submit buttons